src/test/java/uk/gov/pay/ledger/event/EventIntegrationTest.java

```diff
@@ -16,19 +16,20 @@
 @Ignore
 public class EventIntegrationTest {
     @ClassRule
-    public static AppWithPostgresRule rule = new AppWithPostgresRule();
+    public static AppWithPostgresAndSqsRule rule = new AppWithPostgresAndSqsRule();
 
     private Integer port = rule.getAppRule().getLocalPort();
 
     @Test
     public void shouldGetEventFromDB() {
+        var x =rule.getAppRule().getConfiguration();
         Event event = anEventFixture()
-                .insert(rule.getJdbi())
+                .insert(rule.getSqsClient())
                 .toEntity();
 
         given().port(port)
                 .contentType(JSON)
-                .get("/v1/event/" + event.getId())
+                .get("/v1/event/1")
```

Comment:
Not sure why this is `1` ? It will cause flakiness if multiple tests insert events

Reply:
because it uses id generated by db not event id

Reply:
`id` can be anything as other tests may also insert events and `1` is not guaranted

Reply:
I separated this test from just queue processing test: `QueueEventProcessingIT`.